environment_configuration_file: "/home/mrs/flightsim/examples/quadrotor_simulator/config/config.yml"
controller_after_following: "Se3Controller"
